### DISCUSSION TITLE
Functions on watchOS via SPM

### DISCUSSION THREAD

**Discussion by hohteri** [firebase/firebase-ios-sdk#8861] [General]
Hi all,

Based on release notes Functions should have watchOS support since 8.7.0 but SPM doesn't provide it due to Package.swift FirebaseFunctionsTarget having condition for only iOS, macOS and tvOS. Is there a reason for this or is it an oversight?

Thanks,
Harri

**Comment by ncooke3** [comment]
Hi @hohteri, thanks for pointing this out! 

Functions on watchOS via SPM was enabled in #8864 and will be available in the next release (v8.9)!
